src/Command/Patch/Manager.php

```diff
@@ -0,0 +1,185 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\CloudPatches\Command\Patch;
+
+use Illuminate\Contracts\Filesystem\FileNotFoundException;
+use Illuminate\Filesystem\Filesystem;
+use Magento\CloudPatches\Command\Apply;
+use Magento\CloudPatches\Filesystem\DirectoryList;
+use Magento\CloudPatches\Filesystem\FileList;
+use Magento\CloudPatches\Patch\Applier;
+use Magento\CloudPatches\Patch\ApplierException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Manages patches appliance.
+ */
+class Manager
+{
+    /**
+     * Directory for hot-fixes.
+     */
+    const HOT_FIXES_DIR = 'm2-hotfixes';
+
+    /**
+     * @var Applier
+     */
+    private $applier;
+
+    /**
+     * @var Filesystem
+     */
+    private $filesystem;
+
+    /**
+     * @var FileList
+     */
+    private $fileList;
+
+    /**
+     * @var DirectoryList
+     */
+    private $directoryList;
+
+    /**
+     * @param Applier $applier
+     * @param Filesystem $filesystem
+     * @param FileList $fileList
+     * @param DirectoryList $directoryList
+     */
+    public function __construct(
+        Applier $applier,
+        Filesystem $filesystem,
+        FileList $fileList,
+        DirectoryList $directoryList
+    ) {
+        $this->applier = $applier;
+        $this->filesystem = $filesystem;
+        $this->fileList = $fileList;
+        $this->directoryList = $directoryList;
+    }
+
+    /**
+     * Copying static file endpoint.
+     * This resolves issue MAGECLOUD-314.
+     *
+     * @param OutputInterface $output
+     */
+    public function copyStaticFile(OutputInterface $output)
```

Comment:
Do we need this behavior here? It's not patch-related.

Reply:
You're right. Removed